server/core/store.go

```diff
@@ -302,6 +302,15 @@ func (s *StoresInfo) GetStore(storeID uint64) *StoreInfo {
 	return store.Clone()
 }
 
+// ZoreGetStore return a StoreInfo with storeID
+func (s *StoresInfo) ZoreGetStore(storeID uint64) *StoreInfo {
```

Comment:
what does Zore mean?

Reply:
:open_mouth: mean do not clone in here, has changed the name.